### PR TITLE
Integrating spectrum Toast in milo required by Event creation console.

### PR DESCRIPTION
**Pull request by gbajaj91**
Adding SP Toast to milo repo
Adding sp-chevron-100 required by picker.

Resolves: [MWPW-148320](https://jira.corp.adobe.com/browse/MWPW-148320)

Screenshot 2024-06-25 at 9 51 08 AM

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://ecc--milo--gbajaj91.hlx.page/?martech=off

### CODE REVIEW

**Review by yesil — APPROVED** [review]

File: libs/features/spectrum-web-components/src/toast.js [hunk 1]
```diff
@@ -0,0 +1 @@
+import '@spectrum-web-components/toast/sp-toast.js';
```
Comment (3ch023) [inline libs/features/spectrum-web-components/src/toast.js:1]: github reports missing line break

**Review by 3ch023 — APPROVED** [review]

**Comment by yesil** [comment]
@gbajaj91 the catalog page looks okay w.r.t icons but the text block at the top is not, but it is because you have old version of the text block(potentially, other blocks as well).

https://main--cc--adobecom.hlx.live/products/catalog?martech=off&georouting=off&milolibs=ecc--milo--gbajaj91

usually, to overcome this, I push latest milo to my stage/main branches and re-create/sync my feature branch.

**Comment by gbajaj91** [comment]
@yesil resolved by sync latest stage changes.
image

**Comment by yesil** [comment]
Adding read-for-stage, since this has no impact on existing pages.

**Pull request merged**